<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdManagerClass</class>
 <widget class="QMainWindow" name="AdManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="devList">
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
      <property name="batchSize">
       <number>98</number>
      </property>
      <item>
       <property name="text">
        <string>Устройство 1</string>
       </property>
       <property name="icon">
        <iconset resource="AdManager.qrc">
         <normaloff>:/icons_large/icons/large/tablet-lime.png</normaloff>:/icons_large/icons/large/tablet-lime.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Устройство 2</string>
       </property>
       <property name="icon">
        <iconset resource="AdManager.qrc">
         <normaloff>:/icons_large/icons/large/tablet-lime.png</normaloff>:/icons_large/icons/large/tablet-lime.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="actSettings"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Правка</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>&amp;Устройство</string>
    </property>
    <widget class="QMenu" name="menuPlayList">
     <property name="title">
      <string>&amp;Список воспроизведения</string>
     </property>
     <addaction name="actPlayListManage"/>
     <addaction name="actCustomPlaylist"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actUpdateAll"/>
    <addaction name="separator"/>
    <addaction name="actDevStart"/>
    <addaction name="actDevStop"/>
    <addaction name="actDevPause"/>
    <addaction name="actDevRestart"/>
    <addaction name="separator"/>
    <addaction name="actDevSettings_2"/>
    <addaction name="separator"/>
    <addaction name="menuPlayList"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDevice"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actExit">
   <property name="text">
    <string>Вы&amp;ход</string>
   </property>
   <property name="statusTip">
    <string>Выход из программы администрирования</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actUpdateAll">
   <property name="text">
    <string>Обновить список</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actDevStart">
   <property name="text">
    <string>&amp;Запустить</string>
   </property>
   <property name="statusTip">
    <string>Запустить воспроизведение контента на устройстве</string>
   </property>
  </action>
  <action name="actDevStop">
   <property name="text">
    <string>&amp;Остановить</string>
   </property>
   <property name="statusTip">
    <string>Остановить воспроизведение контента на устройстве</string>
   </property>
  </action>
  <action name="actDevPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Пауза</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actDevRestart">
   <property name="text">
    <string>Перезапус&amp;к</string>
   </property>
   <property name="statusTip">
    <string>Начать с начала воспроизведение контента на устройстве</string>
   </property>
  </action>
  <action name="actPlayListManage">
   <property name="text">
    <string>&amp;Управление списками...</string>
   </property>
  </action>
  <action name="actSettings">
   <property name="text">
    <string>Настройки...</string>
   </property>
  </action>
  <action name="actCustomPlaylist">
   <property name="text">
    <string>Изменить...</string>
   </property>
  </action>
  <action name="actDevSettings_2">
   <property name="text">
    <string>&amp;Настройки устройства...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AdManager.qrc"/>
 </resources>
 <connections/>
</ui>
